relay-server/src/services/processor/ourlog.rs

```diff
@@ -69,6 +63,11 @@ pub fn process(
             _ => return ItemAction::Keep,
         };
 
+        if let Err(e) = scrub(&mut annotated_log, &project_info.config) {
+            relay_log::debug!("failed to scrub pii from log: {}", e);
```

Comment:
I think we can upgrade this to an error (for which we will get a sentry error).

```suggestion
            relay_log::error!("failed to scrub pii from log: {}", e);
```